server/controller/user/orderRequest.js

```diff
@@ -0,0 +1,25 @@
+const { addOrderRequest } = require('../../database/queries');
+
+const userOrderReguest = async (req, res, next) => {
+  try {
+    const {
+      body: { providerId, description },
+      user: { id },
+    } = req;
+
+    await addOrderRequest({
+      providerId,
+      description,
+      id,
+    });
```

Comment:
if you want to use all the body so why not pass it all to the query
```suggestion
    const { id }  = req.user;

    await addOrderRequest({ ...req.body, id });
```